linux_os/guide/system/permissions/files/permissions_important_account_files/file_etc_security_opasswd/bash/shared.sh

```diff
@@ -0,0 +1,7 @@
+# platform = multi_platform_sle
+
+# Create /etc/security/opasswd if needed
+# Owner group mode root.root 0600
+[ -f  /etc/security/opasswd ] || touch /etc/security/opasswd
+chown root.root /etc/security/opasswd
```

Comment:
The user-group separator is `:` already for some time, and [usage of dot is discouraged](https://www.gnu.org/software/coreutils/manual/html_node/chown-invocation.html).